lms/envs/common.py

```diff
@@ -1327,9 +1327,29 @@ def _make_locale_paths(settings):
 ]
 
 # Edly Configuration
-EDLY_PANEL_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_RESTRICTED_USERS = 'Edly Panel Restricted Users'
+EDLY_PANEL_USERS_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_ADMIN_USERS_GROUP_NAME = 'Edly Panel Admin Users'
+WORDPRESS_SUBSCRIBER_USERS_GROUP_NAME = 'WordPress Subscriber Users'
+WORDPRESS_EDLY_ADMIN_USERS_GROUP_NAME = 'WordPress Edly Admin Users'
 ADMIN_CONFIGURATION_USERS_GROUP_NAME = 'Admin Configuration Users'
 
+EDLY_ROLES = {
+    'panel_restricted': EDLY_PANEL_RESTRICTED_USERS,
+    'panel_user': EDLY_PANEL_USERS_GROUP_NAME,
+    'panel_admin': EDLY_PANEL_ADMIN_USERS_GROUP_NAME,
+    'subscriber': WORDPRESS_SUBSCRIBER_USERS_GROUP_NAME,
+    'edly_admin': WORDPRESS_EDLY_ADMIN_USERS_GROUP_NAME
+}
+
+EDLY_ROLES_LIST = [
```

Comment:
Why do we need this list when we have `EDLY_USER_ROLES` dictionary above?

Reply:
This was duplicated, which is now code removed.